src/validations/proposal.validation.js

```
@@ -2,29 +2,74 @@ const Joi = require('joi');
+  params: Joi.object().keys({
+    proposalId: Joi.string().custom(objectId).required(),
   }),
+  body: Joi.object()
+    .keys({
+      status: Joi.string().required().valid('accepted', 'rejected', 'withdrawn').messages({
+        'any.only': 'Status must be accepted, rejected, or withdrawn',
+      }),
+      /* biome-ignore lint/suspicious/noThenProperty: Joi conditionally requires `then` */
+      rejectionReason: Joi.string().max(500).when('status', {
+        is: 'rejected',
+        then: Joi.required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)